### PR TITLE
Permission ctd

### PR DESCRIPTION
**Pull request by michaelmattig**
- [ ] I added an entry to [`CHANGELOG.md`](CHANGELOG.md) if knowledge of this change could be valuable to users.

---

Here is a brief summary of what I did:

<TEXT>

### CODE REVIEW

**Comment by michaelmattig** [comment]
[Admin kann Rechte vergeben](https://trello.com/c/3H9dWVE0/1107-admin-kann-rechte-vergeben)

File: services/src/contexts/simple_context.rs [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
 
+// TODO: rename SimpleAppContext
 #[async_trait]
-pub trait SimpleContext: Context<Session = SimpleSession> {
+pub trait SimpleContext: ApplicationContext<Session = SimpleSession> {
     fn default_session(&self) -> Db<SimpleSession>;
     async fn default_session_ref(&self) -> RwLockReadGuard<SimpleSession>;
     async fn default_session_ref_mut(&self) -> RwLockWriteGuard<SimpleSession>;
+
+    async fn default_session_context(&self) -> Self::Context {
```
Comment (jdroenner) [inline services/src/contexts/simple_context.rs:13]: can you add some docs? what does this do?

File: services/src/contexts/in_memory.rs [hunk 2]
```diff
@@ -61,25 +61,29 @@ impl InMemoryContext {
...
         query_ctx_chunk_size: ChunkByteSize,
     ) -> Self {
-        let ctx = Self {
+        let session = SimpleSession::default();
+
+        let app_ctx = Self {
             db: Default::default(),
             task_manager: Default::default(),
-            session: Default::default(),
+            session: Arc::new(RwLock::new(session.clone())),
```
[the remark this answers is not in the document]
Comment (michaelmattig) [reply services/src/contexts/in_memory.rs:69]: I didn't change the type. `Default::default()` created the `Arc<RwLock<Session>>` automatically, but I now need it afterwards again so I explicitly set it.

**Review by jdroenner — APPROVED** [review]

**Comment by jdroenner** [comment]
bors merge

**Pull request merged**